src/gui/bounding_box.cc

```diff
@@ -3,19 +3,35 @@
 #include <glog/logging.h>
 
 namespace {
+#if 1
```

Comment:
Do not use `#if 1`. It's unclear why `#else` part is kept and unused.
And it'll be switch the flag if we have it so many.